File: Javascript/FindTheHighestAltitude.js
// Leetcode Probelm - 1732

var largestAltitude = function (gain) {
    let height = 0
    let maxAltitude = 0

    for (const element of gain){
        height += element
        maxAltitude = Math.max(maxAltitude,height)
    }

    return maxAltitude
};

File: Python/FindTheHighestAltitude.py
## Leetcode Problem - 1732

def largestAltitude(gain):
    maxAltitude = 0
    height = 0

    for element in gain:
        height += element
        maxAltitude = max(maxAltitude,height)

    return maxAltitude


gain = [-5,1,5,0,-7]
print(largestAltitude(gain))